src/Package/Util/Header/Version.php

```diff
@@ -90,14 +90,14 @@ public function fileHasVersionMacro($fname)
     public function getVersionFromHeader()
     {
         $headers = glob($this->package->getSourceDir().DIRECTORY_SEPARATOR.'*.h');
-        $pat = ',define\s+'.$this->macroName.'\s+"(.*)",i';
+        $pat = ',define\s+'.$this->macroName.'\s+(".*"|.*\b),i';
```

Comment:
You can't use `\b` in a character class (it's interpreted as a backspace character), any other suggestions?
Can version numbers contain alpha characters? eg `1.2.3beta` or would `[0-9\.]` work?